pkg/platform/types.go

```diff
@@ -153,17 +153,30 @@ type ProjectMeta struct {
 }
 
 type ProjectSpec struct {
-	DisplayName string `json:"displayName,omitempty"` // Deprecated. Will be removed in the next major version release
+
+	// Deprecated. Will be removed in the next minor (1.6.x) version release
+	// Use ProjectMeta.Name instead
+	DisplayName string `json:"displayName,omitempty"`
 	Description string `json:"description,omitempty"`
 }
 
 type ProjectConfig struct {
-	Meta ProjectMeta
-	Spec ProjectSpec
+	Meta ProjectMeta `json:"meta,omitempty"`
+	Spec ProjectSpec `json:"spec,omitempty"`
```

Comment:
?
I don't think omitempty on meta/spec is wanted

Reply:
correct, removed `omitempty`